components/script/jsonLD/src/index.js

```diff
@@ -5,7 +5,10 @@ function ScriptJsonLD({json}) {
   return (
     <>
       {json && (
-        <script type="application/ld+json">{JSON.stringify(json)}</script>
+        <script
+          dangerouslySetInnerHTML={{__html: JSON.stringify(json)}}
```

Comment:
We have **[htmlStringToReactElement](https://github.com/SUI-Components/sui/tree/master/packages/sui-js#string)**

```js
import {htmlStringToReactElement} from '@s-ui/js/lib/string'

htmlStringToReactElement('<p>No more dangerouslySetInnerHTML</p>')
```

Reply:
it is not HTML string